react/week2/NewToDo.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import AddToDo from './AddToDo';
+
+//bootstrap styling
+import { Table, Button, Badge } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+export default function NewToDo() {
+  const [todos, setToDo] = useState([
+    'brush teeth',
+    'go for walk',
+    'take breakfast',
+    'get ready to office',
+    'project meeting',
+    'client meeting',
+  ]);
+  const [mappedToDos, setMap] = useState();
+
+  const linethrough = {
+    textDecoration: 'line-through',
+    color: 'red',
+  };
+  const centerAlign = {
+    marginRight: '20%',
+    width: '50%',
+  };
```

Comment:
Try to keep styles in a CSS file, avoid inline styles and use classes instead :)